source/Irrlicht/CGLTFMeshFileLoader.cpp

```diff
@@ -77,12 +84,18 @@ CGLTFMeshFileLoader::CGLTFMeshFileLoader() noexcept
 {
 }
 
+/**
+ * The most basic portion of the code base. This tells irllicht if this is a gltf file.
+*/

```

Comment:
Not quite accurate. This method tells Irrlicht if the file has the ".glTF" extension.